three.c

```diff
@@ -0,0 +1,56 @@
+//Albert Zhang
+//Checks what type the triangle is and whether it could be a triangle
+#include <stdio.h>
+#include <stdlib.h>
+char sanitizedInput(int a) {
+	float boi;
+	printf("Input #%d \n", a);
+	scanf("%f", &boi);
+	if (getchar() != '\n') {
+		sanitizedInput(a);
+	}
+	else {
+		return boi;
+	}
+}
+int isTriangle(float q, float r, float s) {
```

Comment:
you don't handle the input gracefully! also, make sure to include comments for each function.